src/__tests__/__snapshots__/main-test.js.snap

```diff
@@ -267,6 +267,35 @@ Object {
   "displayName": "MyComponent",
   "methods": Array [],
   "props": Object {
+    "OtherComponentProps": Object {
```

Comment:
Even though my getFlowType test is working, these are still being put under props in the snapshot. Trying to track this down.